docs/how-ubuntu-is-made/concepts/version-strings.md

```diff
@@ -158,15 +167,15 @@ package used so far.
 
 List of this and further related examples:
 
-| Previous version                        | No-change rebuild in devel   |
-| --------------------------------------- | ---------------------------- |
-| `2.0-2`                                 | `2.0-2build1`                |
-| `2.0-2ubuntu2`                          | `2.0-2ubuntu3`               |
-| `2.0-2build1`                           | `2.0-2build2`                |
-| `2.0ubuntu0` (native in Ubuntu)         | `3.0ubuntu0` or `2.1ubuntu0` |
-| `2.0-0ubuntu1` (version only in Ubuntu) | `2.0-0ubuntu2`               |
-| `2.0` (native in Debian)                | `2.0build1`                  |
-| `2`   (native in Debian)                | `2build1`                    |
+| Previous version                        | No-change rebuild in devel        |
+| --------------------------------------- | --------------------------------- |
+| `2.0-2`                                 | `2.0-2build1`                     |
+| `2.0-2ubuntu2`                          | `2.0-2ubuntu3`                    |
+| `2.0-2build1`                           | `2.0-2build2`                     |
+| `2.0-0ubuntu1` (version only in Ubuntu) | `2.0-0ubuntu2`                    |
+| `2.0ubuntu` (native in Ubuntu)          | `2.0ubuntubuild1` or `2.1ubuntu`  |
```

Comment:
I personally hate to see `2.0ubuntubuild1` - did anyone in the discussion insist on that?
If yes, so be it.
But if not, could that just be `2.1ubuntu` please?